docker/Dockerfile

```diff
@@ -105,7 +106,14 @@ RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest
 
 # Add code stub last
 COPY . /root/code/garage
+# Prevent garage from exiting on config
+RUN cp /root/code/garage/garage/config_personal_template.py \
```

Comment:
Why is this necessary here but not in the headless version?

Reply:
It is necessary in the headless (CI) version as well, but the config files is copied in [Travis](https://github.com/rlworkgroup/garage/blob/77714c38d5b575a5cfd6d1e42f0a045eebbe3484/.travis.yml#L28), and this won't work for local builds.
I forgot to remove the line from the .travis.yml though, so I will do that.

Reply:
what if i am an interactive user and i actually customized my config_personal.py though?

Reply:
i think this logic needs to be moved to the Makefile